src/pages/LogInWithShortLivedTokenPage.js

```diff
@@ -57,19 +57,20 @@ class LogInWithShortLivedTokenPage extends Component {
         const email = lodashGet(this.props.route.params, 'email', '');
         const shortLivedToken = lodashGet(this.props.route.params, 'shortLivedToken', '');
 
-        // User is trying to transition with a different account than the one they are currently signed in as so we will sign out and then sign in with the new authToken
-        if (email !== this.props.session.email) {
-            Session.signOut();
+        const signedIn = this.props.session && this.props.session.authToken;
+        if (!signedIn) {
             Session.signInWithShortLivedToken(accountID, email, shortLivedToken);
             return;
         }
 
-        // If the user is revisiting the component authenticated with the right account, we don't need to do anything, the componentWillUpdate when betas are loaded and redirect
```

Comment:
Actually I just realized this, could you also get rid of the `betas` in this file? This is the last comment explaining why they're here and it's not really relevant to keep the betas in withOnyx, componentDidMount, or propTypes/default.

Reply:
Ok that's a good catch. It doesn't really look like they are used. I'll remove betas and test it out.

Reply:
>the componentWillUpdate when betas are loaded and redirect

I guess that was important because the component stopped updating 😄